syntax/javascript.vim

```diff
@@ -225,7 +225,7 @@ if version >= 508 || !exists("did_javascript_syn_inits")
   HiLink jsComment              Comment
   HiLink jsLineComment          Comment
   HiLink jsEnvComment           PreProc
-  HiLink jsDocComment           Comment
+  HiLink jsBlockComment           Comment
```

Comment:
:triangular_ruler: It would be nice to delete two spaces to preserve column alignment here and on `:56`.